src/test/java/core/basesyntax/service/impl/CsvFileReaderImplTest.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class CsvFileReaderImplTest {
+
+    private static List<String> expected = new ArrayList<>();
```

Comment:
we use this list only in one test, do we need this variable as class level?

Reply:
because otherwise we should initialize it not in the BeforeClass (it will not be seen to the readFromFile_Ok test), but in the test itself. I thought from your previous notes that it is better to instantiate/populate such things in BeforeClass, even if they are used only in one test